src/backend/marsha/core/views.py

```diff
@@ -593,6 +593,46 @@ class DocumentView(BaseLTIView):
     serializer_class = DocumentSerializer
 
 
+@method_decorator(csrf_exempt, name="dispatch")
+@method_decorator(xframe_options_exempt, name="dispatch")
+class LTIConfigView(TemplateResponseMixin, View):
+    """LTI view called to configure Marsha LTI provider."""
+
+    template_name = "core/lti_config.xml"
+    content_type = "text/xml; charset=utf-8"
+
+    # pylint: disable=unused-argument
+    def get(self, request, *args, **kwargs):
+        """Respond to GET request.
+
+        Parameters
+        ----------
+        request : Request
+            passed by Django
+        args : list
+            positional extra arguments
+        kwargs : dictionary
+            keyword extra arguments
+
+        Returns
+        -------
+        XML
+            generated from applying the data to the template
+
+        """
+        return self.render_to_response(
+            {
+                "code": settings.LTI_CONFIG_TITLE.lower(),
```

Comment:
What happen if there is no value for one of this setting ?

Reply:
I'll add tests with empty string and null.